Podfile

```diff
@@ -0,0 +1,237 @@
+        inherit! :search_paths
+    
+        target :StitchDarwinCoreTestUtilsTests do
+            pod 'MongoSwift', '~> 0.0.5'
+    
+            inherit! :search_paths
+        end
+    end
+    
+    target :StitchTwilioService do
+        project 'Darwin/Services/StitchTwilioService/StitchTwilioService.xcodeproj'
+        inherit! :search_paths
+    
+        target :StitchTwilioServiceTests do
+            pod 'MongoSwift', '~> 0.0.5'
+    
+            inherit! :search_paths
+        end
+    end
+    
+    target :StitchRemoteMongoDBService do
+        project 'Darwin/Services/StitchRemoteMongoDBService/StitchRemoteMongoDBService.xcodeproj'
+        inherit! :search_paths
+    
+        target :StitchRemoteMongoDBServiceTests do
+            pod 'MongoSwift', '~> 0.0.5'
+    
+            inherit! :search_paths
+        end
+    end
+    
+    target :StitchLocalMongoDBService do
+        project 'Darwin/Services/StitchLocalMongoDBService/StitchLocalMongoDBService.xcodeproj'
+        pod 'MongoMobile', '~> 0.0.3'
+    
+        target :StitchLocalMongoDBServiceTests do
+            pod 'MongoMobile', '~> 0.0.3'
+    
+            inherit! :search_paths
+        end
+    end
+    
+    target :StitchHTTPService do
+        project 'Darwin/Services/StitchHTTPService/StitchHTTPService.xcodeproj'
+        inherit! :search_paths
+    
+        target :StitchHTTPServiceTests do
+            pod 'MongoSwift', '~> 0.0.5'
+    
+            inherit! :search_paths
+        end
+    end
+    
+    target :StitchFCMService do
+        project 'Darwin/Services/StitchFCMService/StitchFCMService.xcodeproj'
+        inherit! :search_paths
+    
+        target :StitchFCMServiceTests do
+            pod 'MongoSwift', '~> 0.0.5'
+    
+            inherit! :search_paths
+        end
+    end
+    
+    target :StitchAWSSESService do
+        project 'Darwin/Services/StitchAWSSESService/StitchAWSSESService.xcodeproj'
+        inherit! :search_paths
+    
+        target :StitchAWSSESServiceTests do
+            pod 'MongoSwift', '~> 0.0.5'
+    
+            inherit! :search_paths
+        end
+    end
+    
+    target :StitchAWSS3Service do
+        project 'Darwin/Services/StitchAWSS3Service/StitchAWSS3Service.xcodeproj'
+        inherit! :search_paths
+    
+        target :StitchAWSS3ServiceTests do
+            pod 'MongoSwift', '~> 0.0.5'
+    
+            inherit! :search_paths
+        end
+    end
+    
+    target :StitchAWSService do
+        project 'Darwin/Services/StitchAWSService/StitchAWSService.xcodeproj'
+        inherit! :search_paths
+    
+        target :StitchAWSServiceTests do
+            pod 'MongoSwift', '~> 0.0.5'
+    
+            inherit! :search_paths
+        end
+    end
+    
+    target :StitchCoreTestUtils do
+        project 'Core/StitchCoreTestUtils/StitchCoreTestUtils.xcodeproj'
+    
+        inherit! :search_paths
+    end    
+end
+
+post_install do |installer|
```

Comment:
[nit q] do we still need this?

Reply:
Looks like they haven't 😞 . Going to uncomment this block out for now.